README.md

```diff
@@ -11,9 +11,25 @@ This is an SDK for AltaPay written in JavaScript. It is written in a way that ma
 
 3- Set environmental variables: JAVA_HOME to your Java environment, ANT_HOME to the directory you uncompressed Ant to, and add ${ANT_HOME}/bin (Unix) or %ANT_HOME%\bin (Windows) to your PATH.
 
-### Build package
+### Install package
 
-Start the build process by going to the repository directory from the terminal and run:
+**Method 1**
+
+Clone the current repository for the build process.
+
+``` git clone https://github.com/AltaPay/sdk-js.git```
+
+**Method 2**
+
+Install the package via npm
+
+``` npm i altapay-js-api ```
+
+### How to build
+
+Run below command to build the package
+
+> If you install the dependency using Method 2 then you have to run below command in `node_modules/altapay-js-api`
```

Comment:
This should be part of the **Method 1** block right? Since build package is only used for including this as a dependency.